modules/testsuite/pom.xml

```diff
@@ -19,7 +19,7 @@
     <surefire.memory.args>-Xmx640m</surefire.memory.args>
     <surefire.jdwp.args>-Xdebug -Xnoagent -Djava.compiler=NONE -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=5005</surefire.jdwp.args>
     <surefire.management.args>-Dcom.sun.management.jmxremote</surefire.management.args>
-    <additionalGroovyScriptSuffix />
+    <additionalGroovyScriptSuffix>-elytron</additionalGroovyScriptSuffix>
```

Comment:
can't we just get rid of this suffix property by removing it from files too when there is only elytron now?

Reply:
Done